strax/plugins/plugin.py

```diff
@@ -196,9 +196,6 @@ def __copy__(self, _deep_copy=False):
                 plugin_copy.__setattr__(attribute, copy(source_value))
         return plugin_copy
 
-    def __deepcopy__(self):
```

Comment:
This is essentially a bug. `__deepcopy__` never works because `deepcopy(plugin.deps)` will also call `__deepcopy__`. `plugin.deps` is a dictionary of plugins.